src/backend/utils/compatibility_layers.ts

```diff
@@ -597,9 +597,9 @@ export async function initializeCompatibilityLayer() {
     initializationTasks.push(downloadDefaultWine())
   }
 
-  if (isMac) {
+  if (isMac || isLinux) {
     initializationTasks.push(checkRosettaInstall())
```

Comment:
Won't this check rosetta for linux to now?

Reply:
Not really since it only runs on mac:
```export async function checkRosettaInstall() {
  if (!isMac) {
    return
  }```
  
  But I can improve this check though

Reply:
done